bundles/org.openhab.ui.dashboard/src/main/resources/OH-INF/i18n/dashboard.properties

```diff
@@ -1,28 +0,0 @@
-index.welcome = Welcome to openHAB 2
```

Comment:
The translations for those properties were made on Crowdin is there something to do there?